http://trac.osgeo.org/grass/changeset/51052
Doesn't really help because grass refuses to run with "ERROR: wxGUI does not support wxPython 2.9.4.0 yet."
--- gui/wxpython/gui_core/ghelp.py.orig
+++ gui/wxpython/gui_core/ghelp.py
@@ -88,7 +88,6 @@ class SearchModuleWindow(wx.Panel):
         """!Do layout"""
         sizer = wx.StaticBoxSizer(self.box, wx.HORIZONTAL)
         gridSizer = wx.GridBagSizer(hgap = 3, vgap = 3)
-        gridSizer.AddGrowableCol(1)
         
         gridSizer.Add(item = self.searchBy,
                       flag = wx.ALIGN_CENTER_VERTICAL, pos = (0, 0))
@@ -104,6 +103,8 @@ class SearchModuleWindow(wx.Panel):
             gridSizer.Add(item = self.searchChoice,
                           flag = wx.ALIGN_CENTER_VERTICAL | wx.EXPAND, pos = (row, 0), span = (1, 2))
         
+        gridSizer.AddGrowableCol(1)
+
         sizer.Add(item = gridSizer, proportion = 1)
         
         self.SetSizer(sizer)
